config/nginx.conf.erb

```diff
@@ -41,23 +41,27 @@ http {
     include mime.types;
     default_type application/octet-stream;
     sendfile on;
+    keepalive_disable safari;
 
     server {
         listen <%= ENV["PORT"] %>;
         server_name micromasters.herokuapp.com;
+        keepalive_disable safari;
         return 301 https://micromasters.mit.edu$request_uri;
     }
 
     server {
         listen <%= ENV["PORT"] %>;
         server_name mm.mit.edu;
+        keepalive_disable safari;
```

Comment:
Do we need this for these names that are just going to redirect anyway?

Reply:
Oh, I see @blarghmatey said the same thing in https://github.com/mitodl/micromasters/pull/3671#issuecomment-339981509